RNTester/js/PermissionsExampleAndroid.android.js

```diff
@@ -76,7 +76,7 @@ class PermissionsExample extends React.Component<{}, $FlowFixMeState> {
     );
   }
 
-  _onSelectPermission = (permission: string) => {
+  _onSelectPermission = (permission: string | number) => {
```

Comment:
I'm not sure without trying to test it out myself. What flow error do you get without this change?

Reply:
I get this error...
```js
Cannot create Picker element because string [1] is incompatible with number [2] in the
first argument of property onValueChange.

     RNTester/js/PermissionsExampleAndroid.android.js
     38│     return (
     39│       <View style={styles.container}>
     40│         <Text style={styles.text}>Permission Name:</Text>
     41│         <Picker
     42│           style={styles.picker}
     43│           selectedValue={this.state.permission}
                    // this line causes error
     44│           onValueChange={this._onSelectPermission.bind(this)}>
     45│           <Item
     46│             label={PermissionsAndroid.PERMISSIONS.CAMERA}
     47│             value={PermissionsAndroid.PERMISSIONS.CAMERA}
     48│           />
     49│           <Item
     50│             label={PermissionsAndroid.PERMISSIONS.READ_CALENDAR}
     51│             value={PermissionsAndroid.PERMISSIONS.READ_CALENDAR}
     52│           />
     53│           <Item
     54│             label={PermissionsAndroid.PERMISSIONS.ACCESS_FINE_LOCATION}
     55│             value={PermissionsAndroid.PERMISSIONS.ACCESS_FINE_LOCATION}
     56│           />
     57│         </Picker>
     58│         <TouchableWithoutFeedback onPress={this._checkPermission}>
     59│           <View>
     60│             <Text style={[styles.touchable, styles.text]}>
       :
 [1] 79│   _onSelectPermission = (permission: string) => {
```